content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -41,6 +41,15 @@ The settings file is JSON formatted. All values must be strings.  The example be
 }
 ```
 
+> Note: The JSON file must be valid JSON for the install to work, so it's best to validate it before using for an install.
+
+The easiest way to check the application config is valid JSON would be with `python`, which will be present on most Linux installs:
+
+```
+$ python3 -m json.tool settings.json
+Expecting property name enclosed in double quotes: line 8 column 5 (char 171)
```

Comment:
The example should show the output when it works probably not when there is an error. If you want to show an error output, show both.

Reply:
Fixed 👍